ocaml/xe-cli/newcli.ml

```diff
@@ -305,12 +305,14 @@ let exit_status = ref 1
 
 let with_open_tcp_ssl server f =
   let port = get_xapiport true in
+  let verify_cert = None in
```

Comment:
This usage feels like it should be solved by having trust-on-first-use like in xencenter. Using a store for host certificates in the filesystem would work. This is outside the scope of the current work, though.